anthos-bm-gcp-terraform/resources/login.sh

```diff
@@ -50,8 +50,24 @@ kubectl create clusterrolebinding another-binding \
   --clusterrole cluster-admin --serviceaccount default:edga-sa
```

Comment:
I know it's out of scope; if possible can update
- line 17: `edga-sa` -> `edge-sa`
- line 50 as well to use: ${KSA_NAME} instead of `edga-sa`?